src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -110,13 +125,274 @@ public void cannotDeclareItemLostForAClosedLoan() {
+
+    loansStorageClient.replace(loan.getId(), loanWithoutLostPolicy);
+    assertThat(loansStorageClient.getById(loan.getId()).getJson().getString("lostItemPolicyId"),
+      nullValue());
+
+    declareLostFixtures.declareItemLost(new DeclareItemLostRequestBuilder()
+      .forLoanId(loan.getId()));
+
+    assertThat(getAccountsForLoan(loan.getId()), hasSize(0));
+  }
+
+  @Test
+  public void shouldNotAssignFeesWhenNoOwnerConfigured() {
+    feeFineOwnerFixture.delete(feeFineOwnerFixture.cd1Owner());
+
+    final InventoryItemResource item = itemsFixture.basedUponNod();
+    final IndividualResource loan = loansFixture.checkOutByBarcode(item, usersFixture.charlotte());
+
+    declareLostFixtures.declareItemLost(new DeclareItemLostRequestBuilder()
+      .forLoanId(loan.getId()));
+
+    assertThat(getAccountsForLoan(loan.getId()), hasSize(0));
+  }
+
+  @Test
+  public void cannotDeclareItemLostWhenNoAutomatedLostItemFee() {
+    feeFineTypeFixture.delete(feeFineTypeFixture.lostItemFee());
+
+    final InventoryItemResource item = itemsFixture.basedUponNod();
+    final IndividualResource loan = loansFixture.checkOutByBarcode(item, usersFixture.charlotte());
+
+    final Response response = declareLostFixtures.attemptDeclareItemLost(
+      new DeclareItemLostRequestBuilder()
+        .forLoanId(loan.getId()));
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Expected automated fee of type Lost item fee"),
+      hasParameter("feeFineType", "Lost item fee"))));
+  }
+
+  @Test
+  public void cannotDeclareItemLostWhenNoAutomatedLostItemProcessingFee() {
+    feeFineTypeFixture.delete(feeFineTypeFixture.lostItemProcessingFee());
+
+    final InventoryItemResource item = itemsFixture.basedUponNod();
+    final IndividualResource loan = loansFixture.checkOutByBarcode(item, usersFixture.charlotte());
+
+    final Response response = declareLostFixtures.attemptDeclareItemLost(
+      new DeclareItemLostRequestBuilder()
+        .forLoanId(loan.getId()));
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Expected automated fee of type Lost item processing fee"),
+      hasParameter("feeFineType", "Lost item processing fee"))));
+  }
+
+  @Test
+  @Parameters( {
+    "0",
+    "10.00"
+  })
+  public void shouldNotAssignProcessingFeeIfDisabled(double processingFee) {
+    final LostItemFeePolicyBuilder lostItemPolicy = lostItemFeePoliciesFixture
+      .facultyStandardPolicy()
+      .withName("Declared lost fee test policy")
+      .withChargeAmountItemPatron(false)
+      .withLostItemProcessingFee(processingFee)
+      .withNoChargeAmountItem();
+
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(lostItemPolicy).getId());
+
+    final UUID loan = createLoanAndDeclareItLost();
+
+    assertNoFeeAssignedForLoan(loan);
+  }
+
+  @Test
+  @Parameters( {
+    "null",
+    "0.0"
+  })
+  public void shouldNotAssignItemAnotherCostFeeIfAmountMissing(@Nullable Double itemFee) {
```

Comment:
Why would there be `anotherCost` defined in the lost item fee policy?

Reply:
The `anotherCost` means `set cost` here. BE and UI terminology does not match.

Reply:
I don't understand, why does the lost item fee have a name?

What is the terminology in the UI?

Reply:
Oh, sorry, the test verifies that `Lost item fee` is not assigned when `lostItemPolicy.chargeAmountItem.chargeType ==  anotherCost` and `lostItemPolicy.chargeAmountItem.amount either 0 or null`

Reply:
I've tried out the UI for setting up lost item fee policies and it does seem odd.

How does mod-circulation know what fee fine type to use, is it related to charge type?

Reply:
There are preconfigured automated `Lost item fee` and `Lost item processing fee` which were added in scope of https://issues.folio.org/browse/MODFEE-28.

They are loaded on module init.

Reply:
> They are loaded on module init.

They are defined when mod-feefines is enabled for a tenant?

Does circulation refer to them by name?

Does charge type affect anything in circulation?

Reply:
Yes, they are being looked up by name + `automated==true`.

> Does charge type affect anything in circulation?

Do you mean item/processing fee by 'charge type'? Or overdue fee, lost fee, etc?

Reply:
By `charge type` I refer to the additional property that is included with a set item cost fee amount (that the test define as `anotherCost` in line with the UI).

I'm trying to understand if circulation needs to care about this property at all?

Reply:
I think it should, because we can set `Actual cost` on UI, so the `chargeType` will be `actualCost` in this case. This charge type have to be ignored and fee/fine will be created manually, if I correct understand from the last refinement.

Reply:
I think this quirk of the design should be as hidden as possible from circulation.

I think the applicable fees logic is starting to fit into the strategy pattern, e.g. a class for `LostItemSetCostFee` and `LostItemProcessingFee` (and eventually, maybe `ActualLostItemSetCostFee` depending on what we learn about this area).

I think the repository (or a factory or similar) could hide all of the details of the policy structure from the rest of the system and the `LostItemFeePolicy` itself describe the domain rather the underlying structure.

Does that make sense? 

What do you think?

Reply:
Sounds good. But this PR is already quite big, do you mind if I create a separate PR for this changes?

Reply:
Sure, that is totally fine with me